src/pages/index.tsx

```diff
@@ -48,8 +49,12 @@ const QuestionWrapper = styled.div`
   }
 `;
 
+const extractInitialQueryFromRouter = ({ query }: NextRouter): string =>
```

Comment:
this seems something we might need to use in other routes until there is a fix, what about making it available outside this scope?

Reply:
Done 👍